cloudify_hostpool/rest/service.py

```diff
@@ -0,0 +1,64 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import httplib
+
+from flask import Flask
+from flask import jsonify
+from flask_restful import Api
+
+from cloudify_hostpool import exceptions
+from cloudify_hostpool.rest.backend import rest_backend
+
+app = Flask(__name__)
+api = Api(app)
+backend = rest_backend.RestBackend('test.yaml', 'test.db')
```

Comment:
We're not missing it, it just should be named 'test.yaml' and be in folder where we run gunicorn from. DB will handle itself.